addons/hr_holidays/models/hr_employee.py

```diff
@@ -265,3 +265,34 @@ def action_time_off_dashboard(self):
                 'employee_id': self.ids,
             },
         }
+
+    def get_stress_days(self, start_date, end_date):
+        all_days = {}
+
+        stress_days = self._get_stress_days(start_date, end_date)
+        for stress_day in stress_days:
+            num_days = (stress_day.end_date - stress_day.start_date).days
+            for d in range(num_days + 1):
+                all_days[str(stress_day.start_date + relativedelta(days=d))] = stress_day.color
+
+        return all_days
+
+    def _get_stress_days(self, start_date, end_date):
+        employees = self or self.env.user.employee_ids.filtered(lambda e: e.company_id == self.env.company)
```

Comment:
`user.employee_id` will always be the employee of the current company;

```suggestion
        employees = self or self.env.user.employee_id
```